src/Npgsql/TypeHandlers/CompositeHandlers/CompositeMemberHandlerOfStruct.cs

```diff
@@ -84,7 +84,7 @@ public override async Task Write(TComposite composite, NpgsqlWriteBuffer buffer,
 
             buffer.WriteUInt32(PostgresType.OID);
             await (NullableHandler<TMember>.Exists
-                ? NullableHandler<TMember>.WriteAsync(_handler, _get!(ref composite), buffer, lengthCache, null, async)
+                ? NullableHandler<TMember>.WriteAsync!(_handler, _get!(ref composite), buffer, lengthCache, null, async)
```

Comment:
It's not so good that we add bangs every time we use these. See comment below in NullableHandler.